static/styles/image_upload_widget.css

```diff
@@ -153,10 +157,8 @@
 #user-avatar-source {
     font-size: 1em;
     z-index: 99;
-    a {
-        position: relative;
-        top: 10px;
-    }
```

Comment:
Does this modify the `avatar from gravatar` styling as well?  If so, is that intentional or not?

Reply:
here I removed only the anchor element Selector, the attributes inside it were not deleted and were simply moved to #user-avatar-source . as the  #user-avatar-source was used by only the mentioned (`avatar from gravatar`). and I changed the value of top to 12px.  so I guess it affects the styling as well